packages/react/package.json

```diff
@@ -12,6 +12,10 @@
       "import": "./dist/esm/internal.js",
       "require": "./dist/internal.js"
     },
+    "./internal/primitives-catalog": {
+      "import": "./dist/esm/primitives-catalog.js",
+      "default": null
```

Comment:
Do we have a plan to add CJS export in the future? It is a dual(ESM/CJS) package. This seems deviating from the goal

Reply:
I don't know of any plans to add a CJS export as this is an internal package only being used by studio we would do one if they ever needed it, but I'm not sure if there are any plans otherwise. That being said I could add a CJS export for consistency if that is what we agree should happen.

Reply:
Ok, that makes sense. CJS is not a must